src/backtrack.rs

```diff
@@ -36,7 +36,7 @@ type Bits = u32;
 const BIT_SIZE: usize = 32;
 const MAX_SIZE_BYTES: usize = 256 * (1 << 10); // 256 KB
 
-/// Returns true iff the given regex and input should be executed by this
+/// Returns true if the given regex and input should be executed by this
```

Comment:
Oh -- thanks for teaching me something new, never came across that before! I'll remove that change then!